specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingSavingsPlan.json

```diff
@@ -0,0 +1,1204 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-04-01",
+    "title": "BillingManagementClient",
+    "description": "Billing client provides access to billing resources for Azure subscriptions."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/savingsPlanOrders/{savingsPlanOrderId}": {
+      "get": {
+        "description": "Get a savings plan order.",
+        "operationId": "BillingSavingsPlanOrder_Get",
```

Comment:
Shouldn't this be SavingsPlanOrder_Get ? Why are we adding Billing to everything ?

Reply:
This API is for billing persona vs. what we have in service persona https://github.com/Azure/azure-rest-api-specs/blob/main/specification/billingbenefits/resource-manager/Microsoft.BillingBenefits/stable/2022-11-01/billingbenefits.json#L177. 
I believe this operation ID is used when generating rest api documents in https://learn.microsoft.com/en-us/rest/api/billing/. I can make it something like  "SavingsPlanOrder_GetByBillingAccount" for this billing persona scenario

Reply:
Will take a look at the existing operationIds and follow the same pattern of convention